Arm/rover_arm_system.hpp

```diff
@@ -210,6 +211,19 @@ class RoverArmSystem : public sjsu::common::RoverSystem
   }
 
  private:
+  void VerifyNonZeroes(double & acceleration)
```

Comment:
```suggestion
  void IncrementIfZero(double & acceleration)
```
Think this is a bit more correct. Verify makes me think it's a boolean funciton that return true/false and wouldn't alter the state of the value.